ansible/roles/purdue-rebuilderd-worker-vm/files/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+!/bin/sh
+set -e
+RAM_SIZE=2G
+IMAGE_PATH=/vm/worker.img
+IMAGE_SRC="https://mirror.pkgbuild.com/images/latest/Arch-Linux-x86_64-basic-20211115.39179.qcow2"
+
+if [ ! -f "$IMAGE_PATH" ]
+then
+    echo "No image found, downloading latest worker image"
+    curl "$IMAGE_SRC" -o "$IMAGE_PATH"
+fi
+
+cp /etc/hosts hosts
+echo -e "$(ip -j a | jq -r .[1].addr_info[0].local)\tself" >> hosts
+
+virt-copy-in -a $IMAGE_PATH hosts /etc
```

Comment:
could you also add a line doing virt-copy-in of rebuilderd-worker.conf into the machine ? it could be part of a mount from the host or so. Does it make sense?

Reply:
Yup that makes sense